src/main/java/freiman/citibike/json/StationService.java

```diff
@@ -9,5 +16,8 @@ public interface StationService {
     @GET("/gbfs/en/station_status.json")
     Single<Stations> stationStatus();
 
+    @POST("/")
+    @Headers("Content-Type: application/json")
+    Call<CitibikeResponse> postCitibikeRequest(@Body CitibikeRequest request);
```

Comment:
This should go in it's own Service since it's not the same Service as the other two calls.

I never taught you to use Call. Why are you using it now?